examples/Migration/CampaignManagement/CreateCompleteCampaignGoogleAdsApiOnly.php

```diff
@@ -0,0 +1,485 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Migration\CampaignManagement;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Google\Ads\GoogleAds\Lib\V1\GoogleAdsClient;
+use Google\Ads\GoogleAds\V1\Common\ManualCpc;
+use Google\Ads\GoogleAds\V1\Common\ExpandedTextAdInfo;
+use Google\Ads\GoogleAds\V1\Common\KeywordInfo;
+use Google\Ads\GoogleAds\V1\Enums\AdGroupAdStatusEnum\AdGroupAdStatus;
+use Google\Ads\GoogleAds\V1\Enums\AdGroupCriterionStatusEnum\AdGroupCriterionStatus;
+use Google\Ads\GoogleAds\V1\Enums\AdGroupStatusEnum\AdGroupStatus;
+use Google\Ads\GoogleAds\V1\Enums\AdGroupTypeEnum\AdGroupType;
+use Google\Ads\GoogleAds\V1\Enums\AdvertisingChannelTypeEnum\AdvertisingChannelType;
+use Google\Ads\GoogleAds\V1\Enums\BudgetDeliveryMethodEnum\BudgetDeliveryMethod;
+use Google\Ads\GoogleAds\V1\Enums\CampaignStatusEnum\CampaignStatus;
+use Google\Ads\GoogleAds\V1\Enums\KeywordMatchTypeEnum\KeywordMatchType;
+use Google\Ads\GoogleAds\V1\Resources\Ad;
+use Google\Ads\GoogleAds\V1\Resources\AdGroup;
+use Google\Ads\GoogleAds\V1\Resources\AdGroupCriterion;
+use Google\Ads\GoogleAds\V1\Resources\AdGroupAd;
+use Google\Ads\GoogleAds\V1\Resources\Campaign;
+use Google\Ads\GoogleAds\V1\Resources\Campaign\NetworkSettings;
+use Google\Ads\GoogleAds\V1\Resources\CampaignBudget;
+use Google\Ads\GoogleAds\V1\Services\AdGroupCriterionOperation;
+use Google\Ads\GoogleAds\V1\Services\AdGroupOperation;
+use Google\Ads\GoogleAds\V1\Services\AdGroupAdOperation;
+use Google\Ads\GoogleAds\V1\Services\CampaignBudgetOperation;
+use Google\Ads\GoogleAds\V1\Services\CampaignOperation;
+use Google\Protobuf\BoolValue;
+use Google\Protobuf\Int64Value;
+use Google\Protobuf\StringValue;
+
+/**
+ * This code example is the last in a series of code examples that shows how to create
+ * a Search campign using the AdWords API, and then migrate it to the Google Ads API one
```

Comment:
"campaign" is misspelled. Same for similar place of other files.

Reply:
Done.